doc/topics/releases/fluorine.rst

```diff
@@ -453,6 +453,26 @@ proxy.
 
     no_proxy: [ '127.0.0.1', 'foo.tld' ]
 
+Changes to :py:mod:`slack <salt.engines.slack>` Engine
+======================================================
+
+The output returned to Slack from functions run using this engine is now
+formatted using that function's proper outputter. Earlier releases would format
+the output in YAML for all functions except for when states were run.
+
+Enhancements to :py:mod:`wtmp <salt.beacons.wtmp>` Beacon
+=========================================================
+
+A new key, `action`, has been added to the events fired by this beacon, which
```

Comment:
Double backticks for `code` demarcation? :)

There's some other examples below that also need double backticks instead of single.

Reply:
Good catch. I have been writing a lot of markdown this week and my head was still in that mode. :smile: